pkg/commands/process/settings/rules/ruby/lang/hardcoded_secret.yml

```
@@ -3,15 +3,15 @@ patterns:
       $<NAME> = $<STRING_LITERAL>
     filters:
       - variable: NAME
-        regex: (?i)password|api_?key|secret
+        regex: (?i)(password|api_?key|secret)\\b
```

The below felt too restrictive, but it would bring this case in line with the string variants e.g. `(?i)['"](password|api_?key|secret)['"]`. 
Any thoughts or preferences, @didroe ? 

```suggestion
        regex: \\b(?i)(password|api_?key|secret)\\b
```

---

Resolving for now but still open to any comments 🙏